packages/query/src/configure/configure.module.ts

```diff
@@ -15,7 +15,7 @@ async function ensurePool(poolConfig: PoolConfig): Promise<Pool> {
   try {
     await pgPool.connect();
   } catch (e) {
-    if (JSON.stringify(e.message).includes(CONNECTION_SSL_ERROR_REGEX)) {
+    if (JSON.stringify((e as Error).message).includes(CONNECTION_SSL_ERROR_REGEX)) {
```

Comment:
The pattern we have been using is `} catch (e: any) {`, I think it would be good to retain that pattern